operations/FAS_Loader/FAS_Loader_Fails.md

```diff
@@ -0,0 +1,67 @@
+## FAS LOADER FAILS
+
+**NOTE:** this procedure is only for csm-0.9.x releases.
+
+The FAS loader may fail due to issues with the `repomd.xml` file in Nexus.
+This will show up in the `fw-loader` job logs with the following message: `CRITICAL: Failed to get repomd.xml from repo`
+
+You may first notice the issue if firmware is not present in the FAS image list (`cray fas images list`) after running the FAS loader or if a firmware update reports `failed to find file, trying again soon`
+
+To view the FAS loader logs:
+*(The system only keeps pods available for a short amount time, if the command returns no pods, rerun the FAS loader with the procedure below.)*
+
+>Get the fas loader pod name:

```

Comment:
FAS should be capitalized, or if you are referring to the pod name, it should be fas-loader.

Reply:
Updated